### ISSUE TITLE
Create a debug  parameter to orb-agent

### ISSUE THREAD

**Issue by manrodrigues** [ns1labs/orb#723]
To debug the orb agent, you need a parameter that makes it as verbose as possible.

**Comment by dscabral** [comment]
docker run --net=host \          
-e ORB_CLOUD_ADDRESS=localhost \
-e ORB_CLOUD_MQTT_ID=1e95bee1-0650-49a5-8a6a-802d9aed1f32 \
-e ORB_CLOUD_MQTT_CHANNEL_ID=7a87ab87-9d89-4241-9af4-329888be878a \
-e ORB_CLOUD_MQTT_KEY=f0645177-1cfa-481d-afb7-803d72cc0361 \
-e PKTVISOR_PCAP_IFACE_DEFAULT=mock \
-e ORB_TLS_VERIFY=false \
**-e ORB_DEBUG_ENABLE=true \**
ns1labs/orb-agent:develop **run -d**

**Comment by dscabral** [comment]
Considerar para testes os parametros em negrito
Ao rodar dessa forma você terá logs de debug para o MQTT

**Comment by manrodrigues** [comment]
```
2022-01-14T17:27:56.121Z        DEBUG   agent/heartbeats.go:19  heartbeat
2022-01-14T17:28:56.047Z        INFO    pktvisor/pktvisor.go:230        pktvisor stdout {"log": "[2022-01-14 17:28:56.047] [visor] [info] REQUEST: GET /api/v1/policies/__all/metrics/bucket/1 200"}
2022-01-14T17:28:56.047Z        INFO    pktvisor/pktvisor.go:297        scraped metrics for policy      {"policy": "policy", "policy_id": "5997c35b-0de3-41f5-8d36-fc29ac576550", "payload_size_b": 1570}
2022-01-14T17:28:56.048Z        INFO    pktvisor/pktvisor.go:315        scraped and published metrics   {"topic": "channels/1049b169-6d04-4a54-aa6d-76981e515287/messages/be/pktvisor/m/1", "payload_size_b": 1570, "batch_count": 1}
2022-01-14T17:28:56.121Z        DEBUG   agent/heartbeats.go:19  heartbeat
2022-01-14T17:29:56.048Z        INFO    pktvisor/pktvisor.go:230        pktvisor stdout {"log": "[2022-01-14 17:29:56.048] [visor] [info] REQUEST: GET /api/v1/policies/__all/metrics/bucket/1 200"}
2022-01-14T17:29:56.048Z        INFO    pktvisor/pktvisor.go:297        scraped metrics for policy      {"policy": "policy", "policy_id": "5997c35b-0de3-41f5-8d36-fc29ac576550", "payload_size_b": 1568}
2022-01-14T17:29:56.051Z        INFO    pktvisor/pktvisor.go:315        scraped and published metrics   {"topic": "channels/1049b169-6d04-4a54-aa6d-76981e515287/messages/be/pktvisor/m/1", "payload_size_b": 1568, "batch_count": 1}
2022-01-14T17:29:56.121Z        DEBUG   agent/heartbeats.go:19  heartbeat
2022-01-14T17:30:13.999Z        DEBUG   agent/rpc_from.go:41    Group RPC message from core     {"topic": "channels/1e7d578a-918e-48f3-ab2e-b28ecc76cc3a/messages/fromcore", "payload": "{\"schema_version\":\"1.0\",\"func\":\"agent_policy\",\"payload\":[{\"action\":\"manage\",\"id\":\"639cf8f3-9221-495f-ac39-30ba73310728\",\"dataset_id\":\"64c6330c-95f3-4034-919d-b9edfa46898a\",\"name\":\"policy3\",\"backend\":\"pktvisor\",\"version\":0,\"data\":{\"handlers\":{\"modules\":{\"handler_dhcp_1\":{\"config\":{},\"filter\":{},\"type\":\"dhcp\"}}},\"input\":{\"input_type\":\"pcap\",\"tap\":\"default_pcap\"},\"kind\":\"collection\"}}]}"}
2022-01-14T17:30:13.999Z        INFO    policyMgr/manager.go:50 managing agent policy from core {"action": "manage", "name": "policy3", "dataset": "64c6330c-95f3-4034-919d-b9edfa46898a", "backend": "pktvisor", "id": "639cf8f3-9221-495f-ac39-30ba73310728", "version": 0}
2022-01-14T17:30:13.999Z        DEBUG   pktvisor/pktvisor.go:140        pktvisor policy apply   {"policy_id": "639cf8f3-9221-495f-ac39-30ba73310728", "data": {"handlers":{"modules":{"handler_dhcp_1":{"config":{},"filter":{},"type":"dhcp"}}},"input":{"input_type":"pcap","tap":"default_pcap"},"kind":"collection"}}
2022-01-14T17:30:14.001Z        INFO    pktvisor/pktvisor.go:230        pktvisor stdout {"log": "[2022-01-14 17:30:14.001] [visor] [info] policy [policy3]: parsing"}
2022-01-14T17:30:14.002Z        INFO    pktvisor/pktvisor.go:230        pktvisor stdout {"log": "[2022-01-14 17:30:14.002] [visor] [info] policy [policy3]: instantiating Tap: default_pcap"}
2022-01-14T17:30:14.002Z        INFO    pktvisor/pktvisor.go:230        pktvisor stdout {"log": "[2022-01-14 17:30:14.002] [visor] [info] policy [policy3]: instantiating Handler handler_dhcp_1 of type dhcp"}
2022-01-14T17:30:14.007Z        INFO    pktvisor/pktvisor.go:230        pktvisor stdout {"log": "[2022-01-14 17:30:14.007] [visor] [info] policy [policy3]: starting"}
2022-01-14T17:30:14.007Z        INFO    pktvisor/pktvisor.go:230        pktvisor stdout {"log": "[2022-01-14 17:30:14.007] [visor] [info] policy [policy3]: starting input instance: default_pcap-policy3"}
2022-01-14T17:30:14.081Z        INFO    pktvisor/pktvisor.go:230        pktvisor stdout {"log": "[2022-01-14 17:30:14.081] [visor] [info] policy [policy3]: starting handler instance: policy3-handler_dhcp_1"}
2022-01-14T17:30:14.082Z        INFO    policyMgr/manager.go:146        policy applied successfully     {"policy_id": "639cf8f3-9221-495f-ac39-30ba73310728", "policy_name": "policy3"}
2022-01-14T17:30:14.082Z        DEBUG   agent/heartbeats.go:19  heartbeat
2022-01-14T17:30:14.082Z        INFO    pktvisor/pktvisor.go:230        pktvisor stdout {"log": "[2022-01-14 17:30:14.081] [visor] [info] REQUEST: POST /api/v1/policies 200"}
```
